### PR TITLE
Implement fetching balance for Tezos

### PR DESCRIPTION
**Pull request by dpereskokov**
Fixes: #463

### CODE REVIEW

File: platform/tezos/model.go [hunk 1]
```diff
@@ -48,3 +48,17 @@ type Address struct {
...
+
+type DelegateOptions struct {
+	Setable bool `json:"setable"`
+}
+
+type Account struct {
+	Name          Address         `json:"name"`
+	Manager       Address         `json:"address"`
+	Delegate      DelegateOptions `json:"delegate"`
+	Balance       string          `json:"balance"`
```
Comment (vikmeup) [inline platform/tezos/model.go]: can you only include what's needed? to keep models light

File: platform/tezos/client.go [hunk 2]
```diff
@@ -49,3 +49,11 @@ func (c *Client) GetBlockByNumber(num int64) ([]Tx, error) {
 
 	return list, err
 }
+
+func (c *Client) GetAccount(address string) (result Account, err error) {
+	path := fmt.Sprintf("node_account/%s", address)
+	referer := fmt.Sprint("https://tzscan.io/%s", address)
```
Comment (vikmeup) [inline platform/tezos/client.go]: why is this needed? the rest seems to work without this change

Comment (dpereskokov) [reply platform/tezos/client.go]: This particular api returns 403 without this header. It was tricky part of that integration )

File: platform/tezos/model.go (hunk 1 above)
Comment (dpereskokov) [reply platform/tezos/model.go]: I believe we should describe models to allow us to do easy integrations in the future. Each time you want to know what this API returns you just have to pick in the sources and check. Other way you will be needed to have some pain finding api to trigger, fill it with referrer header to make them work etc.

Comment (vikmeup) [reply platform/tezos/model.go]: in blockchain space most likely your code will break because of the changes you don't expect and to make it easier to maintain many platforms it's easier to keep only what's needed
